doc/cprover-manual/index.md

```diff
@@ -10,23 +10,26 @@
 [Loop Unwinding](cbmc/unwinding/),
 [Assertion Checking](cbmc/assertions/)
 
-## 4. [Goto-Analyzer &ndash; Abstract Interpretation](goto-analyzer/)
+## 4. [Compositional Reasoning using Code Contracts](contracts/)
 
-## 4. [Test Suite Generation](test-suite/)
+## 5. [Goto-Analyzer &ndash; Abstract Interpretation](goto-analyzer/)
 
-## 5. [Program Properties](properties/)
+## 6. [Test Suite Generation](test-suite/)
 
-## 6. Modeling
+## 7. [Program Properties](properties/)
+
+## 8. Modeling
 
 [Nondeterminism](modeling/nondeterminism/),
 [Assumptions](modeling/assumptions/),
 [Pointers](modeling/pointers/),
 [Floating Point](modeling/floating-point/)
+[File-local Functions](static-functions/)
```

Comment:
Please don't add. There needs to be a proper section on modular verification. This is very, very confusing.

Reply:
Will remove, but as with the JBMC part I do wonder where that place shall be.

Reply:
It could become part of a new section on modular verification, which has contract-based verification as a subsection.